docs/ossm/injection/README.md

```diff
@@ -0,0 +1,105 @@
+## Installing the Sidecar
+### Injection
+In order to take advantage of all of Istio’s features, pods in the mesh must be running an Istio sidecar proxy.
+
+The following sections describe automatic Istio sidecar injection in the pod’s namespace. Manual injection using the `istioctl` command is not supported by OpenShift Service Mesh.
+
+When enabled in a pod’s namespace, automatic injection injects the proxy configuration at pod creation time using an admission controller.
+
+### Automatic sidecar injection
+Sidecars can be automatically added to applicable OpenShift pods using a [mutating webhook admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/) provided by Istio.
+
+When you set the `istio-injection=enabled` or `istio.io/rev=<revision>` (depending if you use control plane revisions) label on a namespace, any new pods that are created in that namespace will automatically have a sidecar added to them.
+
+Automatic injection occurs at the pod-level. You won’t see any change to the deployment itself. Instead, you’ll want to check individual pods (via `oc describe`) to see the injected proxy.
+
+#### Controlling the injection policy
+You enabled and disabled injection at the namespace level. Injection can also be controlled on a per-pod basis, by configuring the `sidecar.istio.io/inject` label on a pod:
+| Resource | Label | Enabled value | Disabled value |
+| --- | --- | --- | --- |
+| Namespace | `istio-injection` | `enabled` | `disabled` |
+| Pod | `sidecar.istio.io/inject` | `"true"` | `"false"` |
+
+If you are using control plane revisions, revision specific labels are instead used by a matching `istio.io/rev` label. For example, for a revision named `canary`:
```

Comment:
In our case, I believe we are always using control plane revisions, and this will need to read a bit more like: https://github.com/openshift-service-mesh/sail-operator/pull/145/files#r1795663255. As you can see from the comment, I'm asking for some clarity on how we will need to document this.

Reply:
True, I see how upstream doc could be confusing in context of OSSM. I will use "revision names" instead of non-revision vs. revision based installation.